website/pages/coursework/artificial-intelligence.mdx

```diff
@@ -529,6 +531,26 @@ The following platforms improve the experience of using LLMs in production. They
 - [LangSmith](https://www.langchain.com/langsmith): By the creators of LangChain, LangSmith helps you visualize, debug, and improve your LLM apps.
 - [Humanloop](https://humanloop.com/): A platform that empowers teams to build reliable and performant AI systems by providing monitoring, A/B testing, a collaborative prompt workspace, private data storage, and fine-tuning.
 
+#### Open-source LLMs
+
+Platforms like HuggingFace offer support for popular open-source language models born out of community efforts to recreate existing proprietary LLMs provided by miscellaneous companies via APIs or software. These open-source LLMs are easily [finetunable](https://cs3216.com/coursework/artificial-intelligence#fine-tuning) and 
+lightweight. While not as powerful as commercially available LLMs, they offer a free, discounted, yet surprisingly expressive experience in terms of text generation and reasoning capabilities. Here are a few popular models and associated guides to finetune them:
+
+- LLAMA 2 ([`code`](), [`finetuning guide`](https://towardsdatascience.com/fine-tune-your-own-llama-2-model-in-a-colab-notebook-df9823a04a32), [`paper`](https://scontent-xsp1-1.xx.fbcdn.net/v/t39.2365-6/10000000_662098952474184_2584067087619170692_n.pdf?_nc_cat=105&ccb=1-7&_nc_sid=3c67a6&_nc_ohc=bGkSwJy8Xa4AX9SX9XW&_nc_ht=scontent-xsp1-1.xx&oh=00_AfC2wTMNDOi-1j5BHP7STSfurMRMQQANleVSDe7vcMVBOQ&oe=64FDAEFF))
```

Comment:
Are the `code` links meant to be added later? Otherwise it's better to remove empty links